intro-HPC/ch_account.tex

```diff
@@ -267,6 +283,14 @@ \subsection{Users of the \association}
 to share your Mail and Affiliation with KU Leuven (which actually means the VSC accountpage)
 Select the 'Remember' checkbox and click 'Yes, continue'.
 \fi
+
+\ifbrussel
+ \includegraphics*[width=5.73in, height=4.00in, keepaspectratio=false]{ch2-browser-authenticate-brussel}
```

Comment:
Indeed we should find a solution for this. In theory in many cases we could include an isnstitute specific graphic. (unfortunately I do not know how to do that...)